### DISCUSSION TITLE
Add another Options page

### DISCUSSION THREAD

**Discussion by Iain-g** [flyntwp/flynt#476] [Q&A] [answered]
I've successfully added a couple of components to the Global Options page - I'd now like to move one of those onto its own Options page. I tried the standard [ACF Pro code](https://www.advancedcustomfields.com/resources/acf_add_options_page/) in the main functions.php but it just causes errors. I also tried fiddling with code in lib/utils/options.php but without success - anyone know how to do this?

**Comment by timohubois** [comment, marked as answer]
Hi @Iain-g would suggest to add subpages to the existing two option pages. If you want use them, you can use the **3rd argument** of the `addGlobal` and `addTranslatable` functions, to define the name of the subpage and the rest will be done automatically:

```PHP
/* Create a Global Option subpage */
Options::addGlobal('NameOfTheComponent', [], 'NameOfTheSubpage');

/* Create a Translatable Option subpage */
Options::addTranslatable('NameOfTheComponent', [], 'NameOfTheSubpage');
```

Does this match your expectations?

**Reply by Iain-g** [reply]
Hi @timohubois - many thanks, yes that's working OK - ideally I wanted a top-level one with its own dashicon as its something the user will need quite often, but this works fine, thank you!
